lesson_04.md

```diff
@@ -54,8 +54,8 @@ Students will be able to...
 ## Accommodation/Differentiation
 * Allow students to use whatever resources they need, including instructors, peers, and web searches if necessary, to create their programs.  The goal of this lesson is exploration and exposure, not mastery.
 * For students that are hesitant or unwilling to engage, point out some simple, useful features (like basic drawing and the ![](say.png) block) to get them started.  Remind them that complexity or "coolness" are not important for this lab.
-* If you wish, you can add a second day to this lesson for students to share out their programs.  Work with your classroom teacher and/or school IT staff to determine the best way for students to be able to present.
-    * One simple option is to have all students share their projects with you, and then show each one in turn on the teacher computer/projector.
+* If you wish, you can add a second day to this lesson for students to share out their programs on a volunteer basis.  Work with your classroom teacher and/or school IT staff to determine the best way for students to be able to present.
+    * One simple option is to have all students share their projects with you, and then show each one in turn on the teacher computer/projector of those students that volunteer.
```

Comment:
"... on the teacher computer/projector of those students that volunteer". This sentence doesn't parse. Consider "... is to have volunteering students share ...".